config.json

```diff
@@ -1,11 +1,41 @@
 {
+    "users": {
+        "178851504076095488": {
+            "name": "Brady",
+            "embed_color": "15844367"
+        },
+        "323968599687430145": {
+            "name":  "Carlie",
+            "embed_color": "10181046"
+        },
+        "191384926321508353": {
+            "name": "James",
+            "embed_color": "9807270"
+        },
+        "179051738354024449": {
+            "name": "Derick",
+            "embed_color": "3447003"
+        },
+        "636724139922554893": {
+            "name": "Keeth",
+            "embed_color": "15158332"
+        },
+        "209101464944115713": {
+            "name": "Rob",
+            "embed_color": "15105570"
+        },
+        "422470851435298818": {
+            "name": "Robbot",
+            "embed_color": "16580705"
+        }
+    },
```

Comment:
See comments on `util/Utility.py`.

Embed colors shouldn't be hardcoded here as they are subject to change (as long as we're basing them off of roles, anyway).

A dictionary mapping colloquial names to user IDs makes sense for referring to specific users in code in a readable fashion - use the colloquial name to find the user ID from this dict, then use the ID to get the rest of the user's info.